ImperatorToCK3Tests/MapperTests/CultureMapper/CultureMapperTests.cpp

```diff
@@ -0,0 +1,84 @@
+#include "../../ImperatorToCK3/Source/Mappers/CultureMapper/CultureMapper.h"
+#include "gtest/gtest.h"
+#include <sstream>
+
+TEST(Mappers_CultureMapperTests, nonMatchGivesEmptyOptional)
+{
+	std::stringstream input;
+	input << "link = { ck3 = culture imp = culture }";
+	const mappers::CultureMapper culMapper(input);
+
+	ASSERT_FALSE(culMapper.cultureMatch("nonMatchingCulture", "", 0, ""));
+}
+
+TEST(Mappers_CultureMapperTests, simpleCultureMatches)
+{
+	std::stringstream input;
+	input << "link = { ck3 = culture imp = test }";
+	const mappers::CultureMapper culMapper(input);
+
+	ASSERT_EQ(*culMapper.cultureMatch("test", "", 0, ""), "culture");
```

Comment:
Reverse to and from in all tests. ck2toeu4 has them wrong. "culture" should go first, then what you're testing. Do this for all tests.

Reply:
I think all should be good now :)